lib/generators/decidim/app_generator.rb

```diff
@@ -50,6 +50,10 @@ def database_yml
         template "database.yml.erb", "config/database.yml", force: true
       end
 
+      def decidim_controller
+        template "decidim_controller.rb.erb", "app/controllers/decidim_controller.rb", force: true
```

Comment:
This will collide with our won `Decidim::ApplicationController, and that's not good :confused: